blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+  v1.1.0. Apparently certain GPU cards contain broken OptionROMs which cause
+  DMA errors and display initialization problems. Thus the firmware setup will
+  now contain an option to enable/disable the DMA protection on-demand (with
+  default state being disabled to avoid problems with dGPUs). Additionally an
+  option to keep IOMMU enabled during OS control hand-off (actually during the
+  UEFI Exit Boot Services) has been added. This setting may be especially
+  interesting for people seeking for the most secure configuration of the
+  firmware. But keep in mind, your Windows will not boot if you enable this
+  option! Refer to our [firmware setup documentation](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+  for more details.
+
+- ACPI PCI interrupt routing for CPU PCIe Root Ports
+
+  The CPU PCI Express root ports had no interrupt routing information reported
+  in ACPI. Just a small fix.
+
+- OC Watchdog ACPI device as in MSI firmware
+
+  Original MSI firmware has been reporting an ACPI device describing the
+  overclocking watchdog. This watchdog is being used by overclocking platforms
+  since 6th generation of Intel Core processors. The same ACPI device has been
+  added to Dasharo to keep consistency. Possibly Windows overclocking tools may
+  depend on it.
+
+### Changed
+
+- Updated SMMSTORE driver to upstream version in UEFI Payload
+
+  An update to the driver serving as a backend to write variables to the SPI
+  flash from UEFI Payload. The change was introduced due to problems with
+  variable updates on other Dasharo platform. The advantage of the update is a
+  cleaned up code with better quality.
+
+- [Improved visual comfort in the boot manager](https://github.com/Dasharo/dasharo-issues/issues/286)
+
+  Minor visual improvement reported by one of laptop users unable to
+  distinguish the entries in the Boot Manager boot device selection window. Now
+  the currently selected device is indicated by an arrow and highlighted with
+  blue color.
+
+### Fixed
+
+- [MSI PRO Z690-A WIFI DDR4 doesn't initialize IGP on certain Processors models](https://github.com/Dasharo/dasharo-issues/issues/274)
+
+  This fix is a crucial one in this release. Many users reported non-working
+  on-board displays with different CPUs than those which were tested in 3mdeb
+  lab. Now all Alder Lake-S graphics devices should be initialized properly and
+  give firmware output on the screen.
+
+- [MSI PRO Z690-A WIFI DDR4 with two Video Cards (2x Radeon 5600XT) has issues related to MMIO resource allocation](https://github.com/Dasharo/dasharo-issues/issues/245)
+
+  Quite specific issue with board configuration overloaded with GPU cards in
+  the community. As such configuration is not common, the debugging involved
+  the issue reporter cooperation by flashing custom binaries and providing
+  debug logs from Dasharo firmware. It happened that the memory requested by
```

Comment:
Wonderful example of community collaboration, this is what we value and fight for :)